UserManagement/accounts/domain/models.py

```diff
@@ -0,0 +1,74 @@
+from django.contrib.auth.models import AbstractBaseUser, BaseUserManager, PermissionsMixin
+from django.db import models
+from django.utils import timezone
+
+
+# Custom User Manager
+class CustomUserManager(BaseUserManager):
+    def create_user(self, email, password=None, identity_id=None, **extra_fields):
+        """
+        Creates and saves a User with the given email, identity_id, and password.
+        """
+        if not email:
+            raise ValueError('Users must have an email address')
+        if not identity_id:
+            raise ValueError('Users must have an associated identity_id')
+
+        user = self.model(
+            email=self.normalize_email(email),
+            identity_id=identity_id,
+            **extra_fields
+        )
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
+
+    def create_superuser(self, email, password=None, identity_id=None, **extra_fields):
```

Comment:
What does it mean, 'super user' 
WE don't have supper user in our system, we have admin and instructor

Reply:
superuser is the admin in Django